src/util-file.c

```diff
@@ -159,6 +159,17 @@ int FileForceSha256(void)
     return g_file_force_sha256;
 }
 
+int FileStoreIsEnabled(void)
```

Comment:
this is not needed, you can use `RequiresFeature(FEATURE_OUTPUT_FILESTORE)` in the landlock code